client/a8c-for-agencies/lib/permission.ts

```diff
@@ -98,33 +104,28 @@ export const isPathAllowed = ( pathname: string, agency: Agency | null ) => {
 		return false;
 	}
 
-	// Admins can access all paths
-	const role = agency?.user?.role;
-	// For some cases, the role isn't set, so we default to 'a4a_administrator'
-	// FIXME: This should be removed after the role is set for all users
-	if ( ! role || role === 'a4a_administrator' ) {
-		return true;
-	}
-
 	// Everyone can access the landing page and the overview page
 	if ( [ A4A_LANDING_LINK, A4A_OVERVIEW_LINK ].includes( pathname ) ) {
 		return true;
 	}
 
-	// Check if the user has the required capability to access the current path
-	const capabilities = agency?.user?.capabilities;
+	// Check path permissions against the agency's tier and user capabilities.
+	const agencyFeatures = agency?.tier?.features || [];
+	const userCapabilities = agency?.user?.capabilities || [];
+	const capabilities = [ ...agencyFeatures, ...userCapabilities ];
```

Comment:
I wonder if we should want to keep these separate, to avoid possible collisions in the future. Don't have strong opinion though.

Reply:
Hmm. It's a fair concern. I'll add prefixing to the caps coming from the tier features. Like `a4a_agency_*`.

Reply:
Done. The prefix is `a4a_feature_*`